File: Lista1/Python/Lsala1/Ex1.py
tempo = float(input('Digite o tempo (horas): '))
velocidade = float(input('Digite a velocidade (Km): '))
distancia = tempo * velocidade
litros = distancia / 12
print(f'A velocidade média é: {velocidade}, o tempo gasto da viagem é: {tempo}, a distância percorrida é {distancia}, e a quantidade de litros usados é {litros:.2f}')

File: Lista1/C/Lsala1/Ex1.c
#include <stdio.h>

int main()
{
    float V, T, D, LU;
    printf("Entre com velocidade: ");
    scanf("%f", &V);
    printf("Entre com tempo: ");
    scanf("%f", &T);
    D = V * T;
    LU = D / 12;
    printf("A distância percorrida foi %.2f, ja o tempo gasto foi %.2f, a velocidade foi %.2f e os litros usados foram %.2f",D, T, V, LU);
    return 0;
}

